### PR TITLE
doc: Add write real test section into writing your first test page

### PR DESCRIPTION
**Pull request by lucastfa-toptal**
As described in the title.

### CODE REVIEW

File: pages/getting-started/writing-your-first-test.md [hunk 1]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+
+1. Visitar uma página web.
+2. Consultar um elemento.
+3. Interagir com o elemento.
+4. Verificar o conteúdo da página.
+
+### Passo 1: Visitar uma página web
+
+Primeiro, vamos visitar uma página web. Neste exemplo vamos visitar a aplicação 
+[Kitchen Sink](https://docs.cypress.io/examples/examples/applications.html#Kitchen-Sink)
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:149]: we are missing TODO line here

File: pages/getting-started/writing-your-first-test.md [hunk 2]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+describe('My First Test', () => {
+  it('Visits the Kitchen Sink', () => {
+    cy.visit('https://example.cypress.io')
+  })
+})
+```
+
+Salve o arquivo e retorne ao Cypress Test Runner. Você vai perceber algumas coisas:
+
+1. O [Log de Comandos](https://docs.cypress.io/guides/core-concepts/test-runner.html#Command-Log)
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:166]: we are missing TODO line here

File: pages/getting-started/writing-your-first-test.md [hunk 3]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+agora exibe a nova ação `visit`.
+2. A aplicação Kitchen Sink foi carregada no painel de 
+[pré visualização da aplicação](https://docs.cypress.io/guides/core-concepts/test-runner.html#Overview).
+3. O teste está verde, mesmo não tendo nenhuma asserção.
+4. O `visit` exibe um estado pendente em azul até que a página termine de carregar.
+
+Se essa solicitação tivesse retornado com um código de status diferente de 2xx, como 404 ou 500, 
+ou se houvesse um erro de JavaScript no código do aplicativo, o teste teria falhado.
+
+[Veja o exemplo acima neste vídeo](https://docs.cypress.io/img/snippets/first-test-visit-30fps.mp4).
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:176]: what do you think to make consistent and on every vídeo we can type "Vídeo de exemplo." like on the first video in this section. Makes sense?

File: pages/getting-started/writing-your-first-test.md [hunk 4]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+    cy.contains('type')
+  })
+})
+```
+
+Agora o teste deve exibir o comando `CONTAINS` e ainda estar verde.
+
+Mesmo sem adicionar uma asserção, sabemos que está tudo certo! Isso acontece pois muitos dos comandos
+do Cypress são construídos para falhar caso não encontram o que é esperado. Isso é conhecido como uma
+[Asserção Padrão](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Default-Assertions).
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:218]: we are missing TODO line here

File: pages/getting-started/writing-your-first-test.md [hunk 5]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+describe('My First Test', () => {
+  it('clicks the link "type"', () => {
+    cy.visit('https://example.cypress.io')
+
+    cy.contains('type').click()
+  })
+})
+```
+
+É quase possível ler o teste como uma história! Cypress chama isso de encadeamento e comandos são
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:256]:
```suggestion
É quase possível ler o teste como uma história! O Cypress chama isso de encadeamento e comandos são
```

File: pages/getting-started/writing-your-first-test.md [hunk 6]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+Note também que o painel da aplicação foi atualizado depois do clique, abrindo o link e mostrando a
+página de destino.
+
+Agora podemos fazer uma asserção nessa nova página!
+
+[Vídeo de exemplo do teste acima](https://docs.cypress.io/img/snippets/first-test-click-30fps.mp4)
+
+```markdown
+Você pode ver o IntelliSense em seus arquivos de teste ao adicionar uma única linha de comentário especial.
+Leia sobre isso [aqui](https://docs.cypress.io/guides/tooling/IDE-integration.html#Triple-slash-directives).
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:268]: we're missing TODO line here

File: pages/getting-started/writing-your-first-test.md [hunk 7]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+```
+
+### Passo 4: Fazer uma asserção
+
+WIP
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:273]: I think we can remove this for now and add the HTML code for back to top as we see in the other pages.

File: pages/getting-started/writing-your-first-test.md [hunk 8]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+
+![Teste com falha](https://docs.cypress.io/img/guides/first-test-failing-contains.529e740a.png)
+
+```markdown
+Mensagens de Erro
+
+Tomamos o cuidado de escrever centenas de mensagens de erro personalizadas que tentam explicar claramente
+o que deu errado no Cypress. Nesse caso, o Cypress atingiu o tempo limite tentando encontrar o texto hype
+em toda a página. Para ler mais sobre a exibição do erro, leia sobre
+[Erros de depuração](https://docs.cypress.io/guides/guides/debugging.html#Errors).
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:234]: we're missing TODO line here

File: pages/getting-started/writing-your-first-test.md [hunk 9]
```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+não controla. Por quê?
+
+1. São passíveis de mudanças a qualquer momento, o que pode quebrar os testes.
+2. Podem fazer testes A/B o que torna impossível ter resultados consistentes.
+3. Podem detectar que o usuário é um script e bloquear o acesso (como o Google faz).
+4. Podem ter funcionalidades de segurança habilitadas que previne o Cypress de funcionar corretamente.
+
+O ponto do Cypress é ser uma ferramente de uso diário para construir e testar suas próprias aplicações.
+
+Cypress não é uma ferramenta de automação de propósito geral. É pouco adequado para scripts em sites que
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:191]:
```suggestion
O Cypress não é uma ferramenta de automação de propósito geral. É pouco adequado para scripts em sites que
```

**Review by pedrohyvo — CHANGES_REQUESTED** [review]

File: pages/getting-started/writing-your-first-test.md (hunk 1 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md:150]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 2 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md:168]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 3 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 4 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md:221]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 5 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 6 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md:273]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 7 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 8 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md:238]: Fixed

File: pages/getting-started/writing-your-first-test.md (hunk 9 above)
Comment (lucastfa-toptal) [reply pages/getting-started/writing-your-first-test.md]: Fixed

File: pages/getting-started/writing-your-first-test.md [hunk 10]
```diff
@@ -125,3 +125,151 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+describe('My First Test', () => {
+  it('Visits the Kitchen Sink', () => {
+    cy.visit('https://example.cypress.io')
+  })
+})
+```
+
+Salve o arquivo e retorne ao Cypress Test Runner. Você vai perceber algumas coisas:
+
+[//]: <> (TODO - Adicionar link Command Log quando for traduzido)
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:167]:
```suggestion
[//]: <> (TODO - Adicionar Command Log e Pré visualização links)
```

File: pages/getting-started/writing-your-first-test.md [hunk 11]
```diff
@@ -125,3 +125,151 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+[//]: <> (TODO - Adicionar link Debugging quando for traduzido)
+
+> Tomamos o cuidado de escrever centenas de mensagens de erro personalizadas que tentam explicar claramente
+o que deu errado no Cypress. Nesse caso, o Cypress atingiu o tempo limite tentando encontrar o texto hype
+em toda a página. Para ler mais sobre a exibição do erro, leia sobre
+[Erros de depuração](https://docs.cypress.io/guides/guides/debugging.html#Errors).
+
+Antes de adicionar outro comando - vamos consertar esse teste. Substitua `hype` por `type`.
+
+[Vídeo de exemplo do teste acima](https://docs.cypress.io/img/snippets/first-test-contains-30fps.mp4)
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:241]:
```suggestion
[Vídeo de exemplo](https://docs.cypress.io/img/snippets/first-test-contains-30fps.mp4)
```

File: pages/getting-started/writing-your-first-test.md [hunk 12]
```diff
@@ -125,3 +125,151 @@ Expect vem do [Chai](http://www.chaijs.com/).
...
+
+É quase possível ler o teste como uma história! O Cypress chama isso de encadeamento e comandos são
+encadeados para contruir testes que realmente expressam o que a aplicação faz de forma declarativa.
+
+Note também que o painel da aplicação foi atualizado depois do clique, abrindo o link e mostrando a
+página de destino.
+
+Agora podemos fazer uma asserção nessa nova página!
+
+[Vídeo de exemplo do teste acima](https://docs.cypress.io/img/snippets/first-test-click-30fps.mp4)
```
Comment (pedrohyvo) [inline pages/getting-started/writing-your-first-test.md:266]:
```suggestion
[Vídeo de exemplo](https://docs.cypress.io/img/snippets/first-test-click-30fps.mp4)
```

**Review by pedrohyvo — COMMENTED** [review]
@lucastfa-toptal just left few comments. After that it's ready to be merged

**Pull request merged**